nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/action/SetSystemPropertiesAction.java

```diff
@@ -52,14 +63,37 @@ public Topology getTopology(Map<String, String> options) {
 
     public static class SetSystemPropertyComputation extends AbstractBulkComputation {
 
+        public static final String NOTIFY = "param-notify";
+
         public SetSystemPropertyComputation() {
             super(ACTION_NAME);
         }
 
         @Override
         protected void compute(CoreSession session, List<String> ids, Map<String, Serializable> properties) {
-            ids.forEach(id -> properties.forEach((k, v) -> session.setDocumentSystemProp(new IdRef(id), k, v)));
+            Map<String, Serializable> actualProperties = new HashMap<>(properties);
+            String eventId = (String) actualProperties.remove(NOTIFY);
```

Comment:
We should not allow everyone to send any kind of events in mass, there must be some validation.

Reply:
what kind of check would you think of ?

Reply:
Created https://jira.nuxeo.com/browse/NXP-25993 for this purpose

Reply:
I talked to Alain and we won't restrict actions to a subset of users. The goal of BAF is to be available for everyone. Dealing with mistakes is another problem and should be dealt with using monitoring and management APIs to cancel stuff.